guide/popular-topics/common-questions.md

```diff
@@ -377,3 +377,39 @@ Assuming the process is to be done for the guild the message is sent in.
 ```
 
 </branch>
+
+### How do I check the ping of my bot?
+
+There are several ways to get the **"ping"** of the bot. Two of the ways can be by the **WebSocket heartbeat** or by the **API latency**.
+
+<branch version="11.x">
+
+<!-- eslint-skip -->
+
+The first example gets the websocket heartbeat. The `ping` property can be found on the Client object.
+
+```js
+<message>.channel.send(`My WebSocket heartbeat is ${<client>.ping}ms!`);
+```
+
+</branch>
+<branch version="12.x">
+
+<!-- eslint-skip -->
+
+The first example gets the websocket heartbeat. The `ping` property can be found on the WebSocketManager, accessible by `client.ws`.
+
+```js
+<message>.channel.send(`My heartbeat ping is ${<client>.ws.ping}ms!`);
+```
+
+</branch>
+
+
+The second example is message latency. This calculates the time it takes for the client to **receive the request** and for your bot to **acknowledge the command**. The message sent by the bot is then used to measure **the elapsed time** between the trigger and the bots response in **milliseconds**. This is known as the **round-trip latency**.
```

Comment:
The first sentence is semantically wrong here. This is the time between the API creating the initial message and the new message, meaning the time `API -> Bot, Bot -> API`, hence "round-trip".